program_admin/__init__.py

```diff
@@ -100,6 +114,12 @@ async def fetch_minimum_balance(self, size: int) -> int:
         async with AsyncClient(self.rpc_endpoint) as client:
             return (await client.get_minimum_balance_for_rent_exemption(size)).value
 
+    async def account_exists(self, key: PublicKey) -> bool:
```

Comment:
There is already an `account_exists` function in util.py (it's being imported and used already in this file too)